app/src/androidTest/java/ch/sdp/vibester/activity/AuthenticationActivityTest.kt

```diff
@@ -195,4 +195,13 @@ class AuthenticationActivityTest {
         Intents.intended(IntentMatchers.hasComponent(MyProfileActivity::class.java.name))
     }
 
+    @Test
+    fun correctResetPassword() {
+        val email = "incorrectEmail"
+
+        onView(withId(R.id.username)).perform(ViewActions.typeText(email), closeSoftKeyboard())
+        onView(withId(R.id.forgotPassword)).perform(click())
+
```

Comment:
Extra space here

Reply:
yeah, already addressed this one, since there's no way to test Toast I deleted this test :(